### DISCUSSION TITLE
Do LV2 plugins work on Windows 10 in Audacity?

### DISCUSSION THREAD

**Discussion by werame** [audacity/audacity#2494] [Q&A]
I downloaded a few from https://distrho.sourceforge.io/plugins.php (for instance, the 3-Band EQ from the "Mini-Series" or glBars) but none of them appear to work in their LV2 incarnation. These are Windows LV2 binaries I downloaded, not Linux, by the way. I tried with Audacity 3.1.3 (64-bit) . The LV2 plugins continuously reset to the "new" state in the plugin manager and don't show up in any menu. I mean they report as enabled when I change them to that state, but after not finding them anywhere in the menus, you go back to the plug-in manager, and the LV2 ones are back in the "new" state.

The VST(2) versions from same source work. 

There's no error whatsoever reported in the Audacity log for the LV2 ones.

Any ideas what could be wrong?

**Comment by LWinterberg** [comment]
I know there's some jankiness with LV2 plugins (eg #751, #1723) so it wouldn't surprise me if they just broke in other places, too.
